wallet-server-ui/src/app/app-routing.module.ts

```diff
@@ -1,29 +1,46 @@
-import { NgModule } from '@angular/core'
-import { CommonModule } from '@angular/common'
-import { RouterModule, Routes } from '@angular/router'
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from '@angular/router';
 
-import { AboutComponent } from './component/about/about.component'
-import { BuildAcceptOfferComponent } from './component/build-accept-offer/build-accept-offer.component'
-import { ContactsComponent } from './component/contacts/contacts.component'
-import { ContractsComponent } from './component/contracts/contracts.component'
-import { LoginComponent } from './component/login/login.component'
-import { NetworkComponent } from './component/network/network.component'
-import { OffersComponent } from './component/offers/offers.component'
-import { WalletBalanceComponent } from './component/wallet-balance/wallet-balance.component'
-
-import { AuthGuard } from './guard/auth-guard'
+import { AboutComponent } from './component/about/about.component';
```

Comment:
@Christewart @user411 Diffs like this are what I was alluding to earlier when I was mentioning the way VSCode is formatting my files on save. I'm still trying to figure out whether something is wrong on my end -- or if a lot of our code hasn't been formatted according the specified settings in our `.editorconfig` files and `tsconfig` files.

Reply:
I believe i've seen diffs like this before when i build code as well. I also have the same questions wrt to formatting. Ideally we would run a linter on CI so that the build fails when checking files in. 

From a quick glance I don't believe we do that? https://github.com/bitcoin-s/bitcoin-s-ts/blob/06aa450464a465971afdc59a57c13e332ae2d297/.github/workflows/Compile.yml#L21